lib/internal/abort_controller.js

```diff
@@ -166,8 +185,8 @@ class AbortSignal extends EventTarget {
    * @param {any} [reason]
    * @returns {AbortSignal}
    */
-  static abort(
-    reason = new DOMException('This operation was aborted', 'AbortError')) {
+  static abort(reason) {
+    reason ??= lazyDOMException();
```

Comment:
I think this won’t let users define a `null` abort reason. Any reason to use this syntax over what was before? Also it changes the `.length` property from `0` to `1`

Reply:
`.length` property? didn't get you

Reply:
The function's `.length` property (controller.abort.length) which is typically not important but in web standards like AbortController it is

Reply:
Ah ok got it, it expects the reason param now thats why